sap_hana/datadog_checks/sap_hana/sap_hana.py

```diff
@@ -183,7 +183,9 @@ def query_database_status(self):
             tags.extend(self._tags)
 
             db_status = self.OK if status['status'].lower() == 'yes' else self.CRITICAL
-            message = status['details'] or None
+            message = None
+            if db_status != self.OK and status.get('details'):
+                message = status['details'] or None
```

Comment:
If you are checking status.get('details') in the if condition, makes sense assigning None later? will this assignation be executed anytime?

Reply:
right, it is a leftover from before